lib/cli-engine/cli-engine.js

```diff
@@ -1018,7 +1018,11 @@ class CLIEngine {
             try {
                 return require(formatterPath);
             } catch (ex) {
-                ex.message = `There was a problem loading formatter: ${formatterPath}\nError: ${ex.message}`;
+                if (format === "table" || format === "codeframe") {
+                    ex.message = `The ${format} formatter is no longer part of core ESLint. Install it manually with \`npm install -D ${format}\``;
```

Comment:
I think the install command should be `npm install -D eslint-formatter-${format}` per https://eslint.org/docs/user-guide/command-line-interface#-f-format, should it not?